### ISSUE TITLE
NTR: Jaw mass (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7821]
New term request Jaw mass (elthat_100723012528): Mandible or maxilla  UMLS: 0745480

**Comment by pnrobinson** [comment]
@MickeySegal
I think this is probably a synonym of

[Jaw swelling HP:0030793](https://hpo.jax.org/app/browse/term/HP:0030793)

which is used for cherubism and https://omim.org/entry/619323

or is there a sufficient distinction?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000307
- label: Jaw mass
- parents: [Abnormal jaw morphology (HP:0030791)](https://hpo.jax.org/app/browse/term/HP:0030791)
- pmid: 
- def: A swelling or enlargment localized to the mass. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7821
- synonyms:
